testing/fceux/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Danct12 <danct12@disroot.org>
+# Maintainer: Danct12 <danct12@disroot.org>
+pkgname=fceux
+pkgver=2.2.3
+pkgrel=0
+pkgdesc="Fast and ultra-compatible NES/Famicom emulator"
+url="http://www.fceux.com"
+arch="all"
+license="GPL"
+depends="sdl gd gtk+2.0"
+depends_dev="sdl-dev gd-dev gtk+2.0-dev scons mesa-dev glu-dev"
```

Comment:
I think scons isn't meant to be in the developer dependencies but rather in makedepends